src/lib/components/event/event-summary-table.svelte

```diff
@@ -3,7 +3,7 @@
 
   import EventCategoryFilter from '$lib/components/event/event-category-filter.svelte';
```

Comment:
Looks like the links in the expanded row could use centering
|||
|--|--|
|![Screenshot 2023-08-29 at 3 53 44 PM](https://github.com/temporalio/ui/assets/15069288/58b72ed0-ee46-417b-a066-b729147b85de)|![Screenshot 2023-08-29 at 3 52 47 PM](https://github.com/temporalio/ui/assets/15069288/ab3cdfb7-1d5f-418b-a810-b2663e026219)|